webview-ui/src/manualTest/clusterPropertiesTests.tsx

```diff
@@ -115,6 +116,22 @@ export function getClusterPropertiesScenarios() {
             });
             webview.postGetPropertiesResponse(testClusterInfo);
         }
+
+        async function handleAbortAgentPoolOperation(agentPoolName: string, hasError: boolean) {
+            await new Promise(resolve => setTimeout(resolve, 1000));
+            if (hasError) {
+                webview.postErrorNotification("Sorry, I wasn't quite able to abort the operation.");
+                return;
+            }
+
+            const agentPoolProfile = testClusterInfo.agentPoolProfiles.find(p => p.name === agentPoolName)!;
+            agentPoolProfile.provisioningState = "Canceling";
```

Comment:
In reality, it looks as if provisioning state doesn't change after invoking an 'abort' request, so we can remove this line that changes the state to "Canceling".